smart-contracts/contracts/cl-vault/src/test_helpers.rs

```diff
@@ -241,6 +234,162 @@ pub fn mock_deps_with_querier_with_balance(
     deps
 }
 
+pub fn mock_deps_with_querier_with_balance_with_positions(
+    info: &MessageInfo,
+    balances: &[(&str, &[Coin])],
+    main_position: FullPositionBreakdown,
+    secondary_positions: Vec<FullPositionBreakdown>,
+) -> OwnedDeps<MockStorage, MockApi, QuasarQuerier, Empty> {
+    let mut positions = secondary_positions.clone();
+    positions.extend_from_slice(&[main_position.clone()]);
+
+    let mut deps = OwnedDeps {
```

Comment:
Would be nice to have a mock_deps function similar to the one in cosmwasm_std::testing to make that less verbose.

Reply:
I don't think we can get rid of the verbosity here, because cosmwasm::testing doesn't export a nice constructor for `OwnedDeps`. We could improve the verbosity of test helpers a bit by changing `mock_deps_with_querier_with_balance_with_positions` into a builder that we can call similar. So the final call would be something like
`MockedDeps::new().with_querier().with_balance().with.positions().build()`. I'm not sure whether that's actually worth the effort though

Reply:
I was just thinking about introducing a local function `mock_deps(positions, balances)` or options of the arguments. But rather a detail, so happy if you just close this if it is not relevant enough from your side